From fc80851f11783bc15fef769e0fbb25fdc8a1e1c7 Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Tue, 26 Jan 2021 20:42:57 +0000
Subject: [PATCH 2/2] vapoursynth_alt: use atomic_int for async_pending

Fixes

```c
src/libavformat/vapoursynth_alt.c:57:5: error: address argument to atomic operation must be a pointer to _Atomic type ('int *' invalid)
    atomic_fetch_sub(&vs->async_pending, 1);
    ^                ~~~~~~~~~~~~~~~~~~
```

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 libavformat/vapoursynth_alt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavformat/vapoursynth_alt.c b/libavformat/vapoursynth_alt.c
index f7e3dd8879..ec08f0c2c6 100644
--- a/libavformat/vapoursynth_alt.c
+++ b/libavformat/vapoursynth_alt.c
@@ -40,7 +40,7 @@ typedef struct VSContext {
     const VSVideoInfo *vi;
     VSNodeRef *node;
     VSScript *script;
-    int async_pending;
+    atomic_int async_pending;
     int current_frame;
     int ncpu;
 } VSContext;
-- 
2.25.1

